### PR TITLE
Add FormAuth smoke test to Elytron package and Vert.x http package

### LINKED ISSUE #5689: quarkus.http.auth.form.cookie-name is actually not configurable
Due to a bug in FormAuthenticationMechanism the cookie name for FORM Authentication is not configurable and it is always the default.

### PR DESCRIPTION
**Pull request by Karm**
The class FormAuthCookiesTestCase will receive additional work, mostly cookie value rotation, time out etc.

Fixes #5689

### CODE REVIEW

File: extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/security/FormAuthenticationMechanism.java [hunk 1]
```diff
@@ -61,7 +61,7 @@ public void init(HttpConfiguration httpConfiguration, HttpBuildTimeConfig buildT
             key = httpConfiguration.encryptionKey;
         }
         FormAuthConfig form = buildTimeConfig.auth.form;
-        loginManager = new PersistentLoginManager(key, "quarkus-credential", form.timeout.toMillis(),
+        loginManager = new PersistentLoginManager(key, form.cookieName, form.timeout.toMillis(),
```
Comment (gastaldi) [inline extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/security/FormAuthenticationMechanism.java]: Good catch!

Comment (machi1990) [reply extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/security/FormAuthenticationMechanism.java]: +1

**Review by gastaldi — APPROVED** [review]
LGTM

**Comment by machi1990** [comment]
Merged, thank @Karm @gastaldi

**Pull request merged**